recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "3.5.0" %}
+{% set version = "3.6.0rc0" %}
 
 package:
   name: pyproj
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pyproj/pyproj-{{ version }}.tar.gz
-  sha256: 9859d1591c1863414d875ae0759e72c2cffc01ab989dc64137fbac572cc81bf6
+  sha256: 122fc92dda0e875c985a6a14841c474cd77355e2ccaab9f69abbaf6c54c25aba
 
 build:
-  number: 1
-  skip: true  # [py<38]
+  number: 0
```

Comment:
Probably need to change this to 1 before merging & need to wait for https://github.com/conda-forge/pyproj-feedstock/pull/137 to merge first.